Resources/Prototypes/_Reserve/Entities/Spawners/DeadSquad.yml

```
@@ -0,0 +1,40 @@
+- type: randomHumanoidSettings
+  id: DeadsquadReserve
+  speciesBlacklist: [ Arachnid, Diona, Dwarf, Reptilian, SlimePerson, Vulpkanin, Harpy, Rodentia, Tajaran]
```

губы добавили ещё одно поле в этот тип прототипа
```suggestion
  speciesWhitelist: [Human]
```